build.xml

```diff
@@ -175,16 +175,18 @@
         </exec>
     </target>
 
-    <target name="phpstan-packages" depends="phpstan" description="Performs static analysis of PHP files using PHPStan on all packages.">
+    <target name="phpstan" depends="tests-acceptance-build" description="Performs static analysis of PHP files using PHPStan on all packages.">
```

Comment:
What do you mean by "acceptance-build"? The PHPStan test runs after acceptance tests?

Reply:
No. But it runs after initialization of acceptance tests, because `tests-acceptance-build` generates AcceptanceTesterActions.php that is used in tests.